.github/workflows/ruby.yml

```diff
@@ -15,8 +15,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: ['2.7', '3.0', '3.1', '3.2']
-
+        ruby-version: ['3.1', '3.2', '3.3', '3.4']
```

Comment:
do we need to drop ruby 2.7 (also a breaking change), or could we exclude certain combos like `'2.7','sidekiq_7'`?

Reply:
I can re-add 2.7 to the test matrix.

Reply:
Ruby 2.7 went EOL almost two years ago, and we're not testing 3.0 which went EOL almost a year ago. Do we really need to keep supporting it?